tools/autograd/derivatives.yaml

```diff
@@ -181,6 +181,15 @@
 - name: all.dim(Tensor self, int dim, bool keepdim=False) -> Tensor
   self: not_implemented("all")
 
+- name: acosh(Tensor self) -> Tensor
```

Comment:
So inplace version won't support autograd?
If so can you add an entry here for `self: not_implemented("inplace version of acosh") to make sure we send a nice error to the user.

Reply:
As far as I can tell, the reason why inplace versions of trigonometric functions don't have support for autograd is that they need the value of the Tensor (since we do `self.pow(2)`). If someone can verify this, it will be great! From what I see, I couldn't find any other (inplace version) trigonometric function having support for autograd.

Added `not_implemented` message for inplace versions of all the 3 ops here: https://github.com/pytorch/pytorch/pull/38388/commits/cb7bbe7eb467f6389c5c953e021ce4794621ca65. Thanks @albanD! :)